src/ontology/config/prefixes.csv

```diff
@@ -224,8 +224,15 @@ PLANP,http://purl.obolibrary.org/obo/PLANP_
 CARO,http://purl.obolibrary.org/obo/CARO_
 NPO,http://purl.bioontology.org/ontology/npo#NPO_
 ICD10CM,http://purl.bioontology.org/ontology/ICD10CM/
+ICD10CM2,https://icd.codes/icd10cm/
 ICD10WHO,https://icd.who.int/browse10/2019/en#/
+ICD10WHO2010,http://apps.who.int/classifications/icd10/browse/2010/en#/
 OMIMPS,https://www.omim.org/phenotypicSeries/PS
 OMIM,https://omim.org/entry/
 Orphanet,http://www.orpha.net/ORDO/Orphanet_
-GARD,http://purl.obolibrary.org/obo/GARD_
\ No newline at end of file
+GARD,http://purl.obolibrary.org/obo/GARD_
+MEDRA,http://identifiers.org/meddra/
```

Comment:
## Others
<details><summary>Details</summary>
<p>

I can't remember why I saw the need to add these. I think it was to be consistent with `mondo.sssom.config.yml`, but now I see that it is actually presently _not_ consistent with `prefixes.csv`. This will be made redundant with:
- #146

```csv
MEDRA,http://identifiers.org/meddra/
MESH,http://identifiers.org/mesh/
SCTID,http://identifiers.org/snomedct/
```

</p>
</details>